app/src/main/java/org/ole/planet/myplanet/ui/userprofile/UserProfileFragment.java

```diff
@@ -90,6 +90,7 @@ public void setUpRecyclerView(final HashMap<String, String> map, View v) {
             public void onBindViewHolder(@NonNull RecyclerView.ViewHolder holder, int position) {
                 if (holder instanceof AdapterOtherInfo.ViewHolderOtherInfo) {
                     ((AdapterOtherInfo.ViewHolderOtherInfo) holder).tvTitle.setText(keys.get(position));
+                    ((AdapterOtherInfo.ViewHolderOtherInfo) holder).tvTitle.setVisibility(View.VISIBLE);
```

Comment:
@rrijal53 
This line is redundant.
Remove the line  `tvTitle.setVisibility(View.GONE);`. Instead add the visibility `gone` to `title` in `row_other_info.xml`

```
 public static class ViewHolderOtherInfo extends RecyclerView.ViewHolder {
        TextView tvTitle, tvDescription;

        public ViewHolderOtherInfo(View itemView) {
            super(itemView);
            tvTitle = itemView.findViewById(R.id.tv_title);
            tvDescription = itemView.findViewById(R.id.tv_description);
            tvTitle.setVisibility(View.GONE);
        }
    }
```

Reply:
@yuviii  is it not redundant , but setting visibility from xml is cheaper than setting from java code . Nice catch :)